lib/KCard/index.vue

```diff
@@ -320,7 +320,7 @@
             },
             thumbnailStyles: {
               ...thumbnailCommonStyles,
-              borderRadius: '8px 0 0 8px',
+              borderRadius: '0 0 0 0',
```

Comment:
I think let's just remove this  borderRadius  property  from here . it  will still achieve  same result you are trying to achieve here. or you could simplify it  to `borderRadius: 0,`

Reply:
Do u want me to raise another pr with required changes??